### PR TITLE
{devel}[GCCcore/6.4.0] SQLite v3.19.3

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)
depends on PR#4907

### CODE REVIEW

**Comment by JackPerdue** [comment]
Depends on PR #4907 (trying to annotate for Github)

File: easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# eg. http://www.sqlite.org/2017/sqlite-autoconf-3190300.tar.gz
+source_urls = ['http://www.sqlite.org/2017/']
+version_str = '%%(version_major)s%s00' % ''.join('%02d' %
+              int(x) for x in version.split('.')[1:])
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb]: Please fix this style error:
```
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/s/SQLite/SQLite-3.19.3-GCCcore-6.4.0.eb:28:15: E128 continuation line under-indented for visual indent
```

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
ca014.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b2aca8d19c4b8efe2a0f732e82c4f378 for a full test report.

**Comment by verdurin** [comment]
Looks good.

**Comment by verdurin** [comment]
Merging, thanks @JackPerdue

**Pull request merged**